plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/PluginParser.java

```diff
@@ -79,12 +79,19 @@
                         System.out.println("artifactId is required, skipping...");
                         continue;
                     }
+                    Object groupIdObject = pluginInfo.get("groupId");
+                    String groupId = groupIdObject == null ? null : groupIdObject.toString();
                     Map pluginSource = (Map) pluginInfo.get("source");
+                    String incrementalsVersion = null;
                     Plugin plugin;
                     if (pluginSource == null) {
-                        plugin = new Plugin(name, "latest", null);
+                        plugin = new Plugin(name, "latest", null, null);
                     } else {
                         Object versionObject = pluginSource.get("version");
+                        if (!StringUtils.isEmpty(groupId) && versionObject == null) {
+                            System.out.println("Version must be input for " + name + ". Skipping...");
+                            continue;
```

Comment:
Should this throw an error? I would like to know if my config is wrong and I'm missing a plugin